src/sys/signal.rs

```diff
@@ -490,11 +490,13 @@ pub enum SigevNotify {
 
 /// Used to request asynchronous notification of the completion of certain
 /// events, such as POSIX AIO and timers.
+#[cfg(not(target_os = "openbsd"))]
```

Comment:
Might be nicer than to wrap this in a module and #cfg that module and then a single line importing everything from it instead of having all of these #cfgs.